### DISCUSSION TITLE
Add icon for 010 Editor

### DISCUSSION THREAD

**Discussion by xiaozhuai** [elrumo/macOS_Big_Sur_icons_replacements#1425] [Icon request]
Please add icon for 010 Editor

**Comment by OMGzilla-git** [comment]
Done. It's in my profile, should be posted soon. The original icon that I pulled from the app was only available in a single low resolution, so it won't look great at larger icon sizes, but not horribad either.
https://macosicons.com/#/u/omgzilla

EDIT: actually the edges look awful when blown up. I'm going to try something different here!

Ok, I have a new one up. It's a fair bit better. I pulled the icon from the Windows version of the app, which contained a higher resolution version of the icon
![010 Editor](https://user-images.githubusercontent.com/85633412/207911169-444d0157-160f-41dd-bfd1-339d6d5e3e33.png)
.

**Reply by xiaozhuai** [reply]
Thank you very much bro！
